cyclops-ui/src/components/pages/NewModule/NewModule.tsx

```diff
@@ -1295,7 +1295,11 @@ const NewModule = () => {
                 }
                 htmlType="submit"
                 name="Save"
-                disabled={loadingTemplate || loadingTemplateInitialValues}
+                disabled={
+                  loadingTemplate ||
+                  loadingTemplateInitialValues ||
+                  !template.version
```

Comment:
The version is not required for a template, so this might not work in some cases. Can you update the condition so that at least one of the `template.repo`, `template.path` or `template.version` needs to be set?

Reply:
Updated the condition, please recheck.